frontend/src/views/WheresMyGeneV2/components/Main/index.tsx

```diff
@@ -102,6 +103,9 @@ export default function WheresMyGene(): JSX.Element {
     EMPTY_SET as Set<Tissue>
   );
 
+  const [filteredExpandedTissues, setFilteredExpandedTissues] =
+    useState<TissueType[]>(EMPTY_ARRAY);
+
   //(seve): These useEffects are deceptively simple.
```

Comment:
Why create a new piece of component state when we can just read the filtered tissues from our store inside of the heatmap component?